azure-pipelines.yml

```diff
@@ -159,6 +159,16 @@ stages:
             parameters:
               jdk: '1.11'
 
+      - job: Build_JDK13_Linux
+        timeoutInMinutes: 60
+        displayName: 'Linux JDK12 Build'
+        pool:
+          vmImage: 'Ubuntu 16.04'
+        steps:
+          - template: ci-templates/jvm-build-steps.yaml
+            parameters:
+              jdk: '1.12'
```

Comment:
I don't think this exists on Azure - or at least it didn't last time I checked

Reply:
It is now. I got it to run and had a test failure due to a real issue with JDK 12 in one of our tests.

Reply:
Oh, cool!